src/main/scripts/src/app/shared/perspectives/collections/post-it/post-it-collections-perspective.component.scss

```diff
@@ -142,7 +142,7 @@
   .add-collection {
     display: block;
     text-align: center;
-    margin: 45px 72px 0;
+    margin: 45px 32px 0;
```

Comment:
If you want to make the whole 'post-it' area clickable instead of just the button + text, you can cange the whole line to `padding-top: 45px;`

Reply:
I changed it because `Import data` was breaked into two lines, so it looks weird.

Reply:
Ok then.